src/openqaoa-core/qaoa_components/variational_parameters/annealingparams.py

```diff
@@ -1,25 +1,11 @@
-#   Copyright 2022 Entropica Labs
-#
-#   Licensed under the Apache License, Version 2.0 (the "License");
-#   you may not use this file except in compliance with the License.
-#   You may obtain a copy of the License at
-#
-#       http://www.apache.org/licenses/LICENSE-2.0
-#
-#   Unless required by applicable law or agreed to in writing, software
-#   distributed under the License is distributed on an "AS IS" BASIS,
-#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#   See the License for the specific language governing permissions and
-#   limitations under the License.
-
-from typing import List, Tuple, Union
+from typing import List, Union
```

Comment:
I would call this file 'annealing.py' instead of 'annealingparams.py', the same way as we call 'shortespath.py' instead of 'problem_shortestpath.py'

Reply:
`annealingparams.py` keeps in like with the naming scheme used for the files containing parameterizations and the class names itself. This naming scheme is preferred since its more verbose.